bundle.Dockerfile

```diff
@@ -9,16 +9,3 @@ LABEL operators.operatorframework.io.bundle.channel.default.v1=4.6
 
 COPY bundle/manifests /manifests/
 COPY bundle/metadata /metadata/
-
-LABEL com.redhat.delivery.operator.bundle=true
-LABEL com.redhat.openshift.versions="v4.6"
-
-LABEL \
-    com.redhat.component="elasticsearch-operator" \
-    version="v1.1" \
-    name="elasticsearch-operator" \
-    License="ASL 2.0" \
-    io.k8s.display-name="elasticsearch-operator bundle" \
-    io.k8s.description="bundle for the elasticsearch-operator" \
-    summary="This is the bundle for the elasticsearch-operator" \
-    maintainer="AOS Logging <aos-logging@redhat.com>"
```

Comment:
this is required to stay in there... i think i missed a change to our generate. but can you manually add this back in?